src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,10 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a
+    className="UserInfo"
+    href={`mailto:${user.email}`}
```

Comment:
If the email will have the value undefined, then we will get an error, try to predict it